src/lib/ext-activitylog.js

```diff
@@ -9,7 +9,7 @@ class Model {
       type: new Set(),
       name: new Set(),
       keyword: '',
-      timeStamp: null,
+      timeStamp: {},
```

Comment:
@atiqueahmedziad why you had to set this back to an empty object instead of null? at a first glance this change looks not strictly needed for this PR, would you mind to briefly explain the reason behind this choice?

Reply:
Previously, I made it `null` when we don't have any timestamp filters applied. To handle different types i.e. `object` and `null` in a single variable, I had to [add some extra codes](https://github.com/mozilla/extension-activity-monitor/pull/40/files#diff-0d42cddec555ff90f252f9e7b5f94496L31-L35) which I can avoid if I stick to one type i.e. `object` only. Also, I will have to put another condition [here](https://github.com/mozilla/extension-activity-monitor/pull/40/files#diff-0d42cddec555ff90f252f9e7b5f94496R76-R78) to check if timestamp is an empty object. If it's empty, I will assign `null` to it. And that will add some more extra codes. 

@rpl what do think about this? Should we do keep it `null`?

Reply:
@atiqueahmedziad the link in your comment (https://github.com/mozilla/extension-activity-monitor/pull/40/files#diff-0d42cddec555ff90f252f9e7b5f94496L31-L35) isn't pointing me to that extra code, would you mind to paste me a snippet in a comment so that I can see if there was a way to make that simpler?

Reply:
oh, sorry. I was pointing to the following code which is in L31-L35 in `ext-activitylog.js` file in this PR. 
```
    // this.timestamp is null when timestamp filter is not applied.
    if (!this.timeStamp) {
      this.timeStamp = {};
    }
```
I had to assign `this.timeStamp` to empty object and then I put value to `start` or `stop` property of the `this.timeStamp` object.